libs/data-access/falconer/src/lib/islamic/get-disclaimer.ts

```diff
@@ -0,0 +1,24 @@
+import { FALCONER_ENDPOINT } from '../../constants';
+import { FalconerRequestInit } from '../../types';
+
+export async function getIslamicDisclaimer(
+  options: Partial<FalconerRequestInit>,
+  locale: string,
+) {
+  const requestUrl = new URL(
+    `/islamic/disclaimer?locale=${locale}`,
+    FALCONER_ENDPOINT,
+  );
```

Comment:
```suggestion
  const requestUrl = new URL(
    '/islamic/disclaimer',
    FALCONER_ENDPOINT,
  );
  requestUrl.searchParams.append('locale', locale);
```

Reply:
here and everywhere in this pr where you request localised data

Reply:
Done